packages/iov-bns/scripts/jsonbytes

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Usage: jsonbytes <filename> <jq selector to field>
+# Extracts the bytes (base64 encoded) and converts to hex
+
+cat "$1" | jq -r "$2" | base64 -d - | cat - | od -x | cut -c9- | tr -d ' \n'
```

Comment:
1. Unnecessary cat: `jq -r "$2" "$1"`
1. Does `base64 -d` work on Linux? On mac it is `-D, --decode   decodes input`
2. `cat -` is a no-op, right?
3. I think the byte order is not correct in `| od -x | cut -c9-`. Confirm `echo abc | xxd -p` vs. `echo abc | od -x | cut -c9-`. You can use `xxd -p -c 999999999` to avoid line breaks.

Reply:
wow!
`I think the byte order is not correct in | od -x | cut -c9-. Confirm echo abc | xxd -p vs. echo abc | od -x | cut -c9-. You can use xxd -p -c 999999999 to avoid line breaks.`

I will fix scripts.

Thank you so much!

Reply:
2. `base64 -d` is linux, `base64 -D` for osx :disappointed: 
Can you see if `base64 --decode` also works on osx, as a generic alternative?

Reply:
> Can you see if base64 --decode also works on osx, as a generic alternative?

It is documented, so yeah, it will do the job

Reply:
`xxd -p -c 9999` tip is gold :+1: